change/@microsoft-fast-components-df40004a-0cb5-4dcd-a8de-064e25d80198.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "update markdown files",
```

Comment:
Often, changelogs are used to learn about changes made in terms of bugs resolution or features.  It may be more readable to users if it said something a little more specific. For example, "updated markdown to bring better clarity to index card web component" or something.  Short like you have, but a little more info so they don't have to click to see what changes, rather they can read to get more idea if they should click for detail. It also helps improve SEO when we're more specific for public facing sites and developers.

Reply:
That makes a lot of sense, thank you for the pro-tip @awentzel !